src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -62,7 +62,7 @@ export class LWCEslintStrategy implements EslintStrategy {
 	}
 
 	ruleDefaultEnabled(name: string): boolean {
-		return EslintStrategyHelper.isDefaultEnabled(this.recommendedConfig, name);
+		return EslintStrategyHelper.getDefaultStatus(this.recommendedConfig, name) === RuleDefaultStatus.ENABLED;
```

Comment:
Could this be made consistent with the one above? It took some time to determine if they were different, they appear the same.

Reply:
On it.